docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_server.md

```diff
@@ -0,0 +1,78 @@
+---
+title: CREATE SERVER statement [YSQL]
+headerTitle: CREATE SERVER
+linkTitle: CREATE SERVER
+description: Use the CREATE SERVER statement to create a foreign server.
+menu:
+  latest:
+    identifier: ddl_create_server
+    parent: statements
+aliases:
+  - /latest/api/ysql/commands/ddl_create_server/
+isTocNested: true
+showAsideToc: true
+---
+
+## Synopsis
+
+Use the `CREATE SERVER` command to create a foreign table. 
+
+## Syntax
+
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#grammar" class="nav-link active" id="grammar-tab" data-toggle="tab" role="tab" aria-controls="grammar" aria-selected="true">
+      <i class="fas fa-file-alt" aria-hidden="true"></i>
+      Grammar
+    </a>
+  </li>
+  <li>
+    <a href="#diagram" class="nav-link" id="diagram-tab" data-toggle="tab" role="tab" aria-controls="diagram" aria-selected="false">
+      <i class="fas fa-project-diagram" aria-hidden="true"></i>
+      Diagram
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="grammar" class="tab-pane fade show active" role="tabpanel" aria-labelledby="grammar-tab">
+    {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/create_server.grammar.md" /%}}
+  </div>
+  <div id="diagram" class="tab-pane fade" role="tabpanel" aria-labelledby="diagram-tab">
+    {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/create_server.diagram.md" /%}}
+  </div>
+</div>
+
+## Semantics
+
+Create a foreign server named *server_name*. If *server_name* already exists in the specified database, an error will be raised unless the `IF NOT EXISTS` clause is used.
+
+### Type
+The `TYPE` clause can be optionally used to specify the server type.
+
+### Server Version
+
+The `VERSION` clause can be optionally used to specify the server version.
+
+### FDW name
+The `FOREIGN DATA WRAPPER` clause can be used to specify the name of the foreign-data wrapper.
+
+### Options:
+The `OPTIONS` clause specifies options for the foreign server. They typically define the connection details of the server, but the actual permitted option names and values are specific to the server’s foreign data wrapper. 
+
```

Comment:
Kill multiple-blank lines, replace with single blank. (Markdownlint catches this and more automatically btw)
```suggestion
```